packages/formspree-react/test/useForm.test.js

```diff
@@ -1,25 +1,25 @@
 import React from 'react';
 import { FormspreeProvider, useForm } from '../src';
-import ReactDOM from 'react-dom';
-import ReactTestUtils from 'react-dom/test-utils';
+import ReactDOM from 'react-dom/client';
+import { act, screen, waitFor } from '@testing-library/react';
+import user from '@testing-library/user-event';
```

Comment:
@colevscode seems like there're a lot of React 18 + test library upgrade specifics (IIRC `@testing-library` needs upgraded for React 18). Is it possible to extract those changes to a separate PR?